lib/TPP/Dialect/Tpp/TppUtils.cpp

```diff
@@ -337,10 +337,13 @@ bool canMapToTppRelu(linalg::GenericOp linalgOp) {
 }
 
 // Return true if the linalg.generic can be mapped to a tpp.relu.
+// TODO: check input arguments:
+// 1. Single input, single output as long as the output bbarg is not used
+// 2. Single output.
 bool isTppRelu(linalg::GenericOp linalgOp) {
   if (linalgOp.getNumLoops() != linalgOp.getNumParallelLoops())
     return false;
-  if ((linalgOp.getNumDpsInputs() != 0) || (linalgOp.getNumDpsInits() != 1))
+  if (linalgOp.getNumDpsInits() != 1)
```

Comment:
I'm surprised this wasn't being tested by any test we have. I was expecting such a change to break something...

Reply:
@KavithaTipturMadhu I think this was something you added, can you check if the tests you had are still ok?

Reply:
I didn't add this. I'm not sure i understand what's going on in this change, are dpsinputs never 0?

Reply:
I will fix it on Monday. Just made sure the tests are working. We used to assume to have 1 inputs for relu.

Reply:
Ah, right, I thought this was some VNNI alternative shape, but relu doesn't care, my mistake.